eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/api/EclairApi.scala

```diff
@@ -48,11 +48,15 @@ trait EclairApi {
 
   def channel(id: ChannelId): Future[ChannelResult]
 
-  def connect(nodeURI: NodeUri): Future[String]
+  def channelStats(): Future[Vector[ChannelStats]]
 
-  def connect(nodeId: NodeId, host: String, port: Int): Future[String]
+  def connect(nodeURI: NodeUri): Future[Unit]
 
-  def close(id: ChannelId, spk: ScriptPubKey): Future[String]
+  def connect(nodeId: NodeId, host: String, port: Int): Future[Unit]
+
+  def disconnect(nodeId: NodeId): Future[Unit]
+
+  def close(id: ChannelId, spk: ScriptPubKey): Future[Unit]
```

Comment:
I'm pretty sure `close` returns the transaction ID of the closing transaction, which would be `DoubleSha256DigtestBE`

Reply:
"ok" is not valid TxId